src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
Cool, I like we use the widget notation for smart components

At a later PR I would love to see a dumb dumb one receiving just the order data

Reply:
Yes, I left a TODO to split it later.
Mainly when we start to really wire it up. 
Right now is mostly a placeholder.